<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="17316" started-at="2014-08-15T21:41:04Z" finished-at="2014-08-15T21:41:21Z">
    <groups>
    </groups>
    <test name="FirefoxTest" duration-ms="17316" started-at="2014-08-15T21:41:04Z" finished-at="2014-08-15T21:41:21Z">
      <class name="multibrowser.MultiBrowserTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:multibrowser.MultiBrowserTest@1fbc7afb]" name="beforeTest" is-config="true" duration-ms="11607" started-at="2014-08-15T14:41:04Z" finished-at="2014-08-15T14:41:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="googleLaunch()[pri:0, instance:multibrowser.MultiBrowserTest@1fbc7afb]" name="googleLaunch" duration-ms="5143" started-at="2014-08-15T14:41:16Z" finished-at="2014-08-15T14:41:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- googleLaunch -->
        <test-method status="FAIL" signature="assertion2()[pri:0, instance:multibrowser.MultiBrowserTest@1fbc7afb]" name="assertion2" duration-ms="12" started-at="2014-08-15T14:41:21Z" finished-at="2014-08-15T14:41:21Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Assert.fail2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Assert.fail2
	at org.testng.Assert.fail(Assert.java:94)
	at multibrowser.MultiBrowserTest.assertion2(MultiBrowserTest.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1220)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- assertion2 -->
        <test-method status="PASS" signature="aftertest()[pri:0, instance:multibrowser.MultiBrowserTest@1fbc7afb]" name="aftertest" is-config="true" duration-ms="503" started-at="2014-08-15T14:41:21Z" finished-at="2014-08-15T14:41:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftertest -->
      </class> <!-- multibrowser.MultiBrowserTest -->
    </test> <!-- FirefoxTest -->
  </suite> <!-- Suite -->
  <suite name="AllSuites" duration-ms="0" started-at="2014-08-15T21:41:32Z" finished-at="2014-08-15T21:41:32Z">
    <groups>
    </groups>
  </suite> <!-- AllSuites -->
  <suite name="Failed suite [Failed suite [Failed suite [Default suite]]]" duration-ms="9874" started-at="2014-08-15T21:41:21Z" finished-at="2014-08-15T21:41:31Z">
    <groups>
    </groups>
    <test name="Default test(failed)(failed)(failed)" duration-ms="9874" started-at="2014-08-15T21:41:21Z" finished-at="2014-08-15T21:41:31Z">
      <class name="multibrowser.MultiBrowserTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:multibrowser.MultiBrowserTest@3f3afe78]" name="beforeTest" is-config="true" duration-ms="9394" started-at="2014-08-15T14:41:21Z" finished-at="2014-08-15T14:41:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="assertion2()[pri:0, instance:multibrowser.MultiBrowserTest@3f3afe78]" name="assertion2" duration-ms="3" started-at="2014-08-15T14:41:31Z" finished-at="2014-08-15T14:41:31Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Assert.fail2]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Assert.fail2
	at org.testng.Assert.fail(Assert.java:94)
	at multibrowser.MultiBrowserTest.assertion2(MultiBrowserTest.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1220)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- assertion2 -->
        <test-method status="PASS" signature="aftertest()[pri:0, instance:multibrowser.MultiBrowserTest@3f3afe78]" name="aftertest" is-config="true" duration-ms="472" started-at="2014-08-15T14:41:31Z" finished-at="2014-08-15T14:41:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftertest -->
      </class> <!-- multibrowser.MultiBrowserTest -->
    </test> <!-- Default test(failed)(failed)(failed) -->
  </suite> <!-- Failed suite [Failed suite [Failed suite [Default suite]]] -->
</testng-results>
